js/indicators/keltner-channels.src.js

```diff
@@ -0,0 +1,196 @@
+'use strict';
+
+import H from '../parts/Globals.js';
+import '../parts/Utilities.js';
+import './ema.src.js';
+import './atr.src.js';
```

Comment:
We don't load indicators anymore - user should do this + we have an error when indicator is not found.

Reply:
In MACD we loading it -> https://github.com/highcharts/highcharts/blob/master/js/indicators/macd.src.js#L4

Reply:
Yes, MACD needs this change in v7 too - we will stop doing this. Now multiple indicators use EMA, so we don't want to load more than needed. It will artificially increase packages/bundles sizes.